x/evm/keeper/migrate.go

```diff
@@ -49,6 +61,109 @@ func removeExternalTokenBurnerCode(ctx sdk.Context, ck chainKeeper) error {
 	return nil
 }
 
+func migrateCommandBatchSignature(ctx sdk.Context, ck chainKeeper, signer types.Signer, multisig types.MultisigKeeper) error {
+	commandBatchID := ck.getLatestSignedCommandBatchID(ctx)
+	for {
+		commandBatchMetadata := ck.getCommandBatchMetadata(ctx, commandBatchID)
+		if commandBatchMetadata.ID == nil {
+			break
+		}
+		commandBatchID = commandBatchMetadata.PrevBatchedCommandsID
+
+		// only migrate secondary key
+		tssKey, ok := signer.GetKey(ctx, tss.KeyID(commandBatchMetadata.KeyID))
+		if tssKey.Role != tss.SecondaryKey {
+			continue
+		}
+
+		// only migrate command batch signed by active key
+		key, ok := multisig.GetKey(ctx, commandBatchMetadata.KeyID)
+		if !ok {
+			break
+		}
+
+		if commandBatchMetadata.Status != types.BatchSigned {
+			if commandBatchMetadata.Status == types.BatchSigning {
```

Comment:
If the batch is already aborted, then we want to skip also as there are no signatures.

Reply:
it is already in that way, if status is not signed, we continue